contracts/other/notional/README.md

```diff
@@ -0,0 +1,35 @@
+# Notional fCash as collateral
+
+## About fCash
+Notional borrowing and lending positions are stored as [fCash](https://docs.notional.finance/notional-v2/notional-v2-basics/fcash).
+
+fCash positions can be positive or negative. If positive they are equivalent for a lending position, and are redeemable at maturity for their face value. If negative they are borrowing positions.
+
+We want to use fCash lending positions as collateral. Given that we know that they are redeemable for their face value at maturity and their volatility is low, they are good assets for that purpose.
```

Comment:
Are we fairly certain on this?  Should we consider if they are hacked or something and the face value cannot be redeemed?